gcc/common/config/riscv/riscv-common.cc

```diff
@@ -211,6 +212,7 @@ static const struct riscv_ext_version riscv_ext_version_table[] =
 
   {"xcorev",    ISA_SPEC_CLASS_NONE, 1, 0},
   {"xcorevelw", ISA_SPEC_CLASS_NONE, 1, 0},
+  {"xcorevmac", ISA_SPEC_CLASS_NONE, 1, 0},
```

Comment:
This should be `xcvmac`.    See the  [agreed specification](https://github.com/openhwgroup/core-v-sw/blob/master/specifications/corev-isa-extension-naming.md).